yarn.lock

```diff
@@ -3182,6 +3182,21 @@ atob@^2.1.2:
   resolved "https://registry.verdaccio.org/atob/-/atob-2.1.2.tgz#6d9517eb9e030d2436666651e86bd9f6f13533c9"
   integrity sha512-Wm6ukoaOGJi/73p/cl2GvLjTI5JM1k/O14isD73YML8StrH/7/lRFgmg8nICZgD3bZZvjwCGxtMOD3wWNAu8cg==
 
+aws-sdk@^2.415.0:
```

Comment:
Can be deduped with below aws-sdk

Reply:
Will remove

Reply:
Well it's the library s3-ls@^3.0.0 depends on aws-sdk "^2.415.0". I guess I can not do much here?

Reply:
You can. Delete both aws-sdk entries from yarn.lock, then run `yarn` again. You should end up with an entry like:

```
aws-sdk@^2.415.0, aws-sdk@^2.607.0:
  version "2.612.0"
```

You can also use https://github.com/atlassian/yarn-deduplicate to help out with this.

Reply:
Thanks for the tip. I cloud be be part of the yarn CLI. Anyway all CR are fixed.